### PR TITLE
Create 09.29 SearchView.md

### LINKED ISSUE #5860: Android Kotlin: 09.29 SearchView in android
<hr>

## Description :scroll:

* Give a step-by-step tutorial of SearchView in android (Kotlin).
*  Required Documentation or Video or Audio.

<hr>

## Domain of Contribution :bar_chart:

- [x] Android Dev (Kotlin)

<hr>

<hr>

## References :
https://www.javatpoint.com/android-searchview

<hr>

## Please Read it Carefully Before Contributing
* Changes should be made in Android_Development_With_Kotlin/ directory under Android_Development_With_Kotlin branch.
* Inside Android_Development_With_Kotlin/ directory creates a directory and names it according to the topic ( for this issue 09. Dynamic User Interface)
* Inside 09. Dynamic User Interface/ directory add your task with subtopic number and name and make changes accordingly.
* You have to push your PR to the Android_Development_With_Kotlin branch only.
* You can take help to the references but don't simply try to copy it. Just explain in your own word.
* This issue is for GWOC'21 contributors only.
* You need to push in a single commit. If there exists more than one commit then you need to squash your commits into one commit.
* Will assign you on a first come first serve basis.
* Please try to complete your assigned task within 7 days. If you need some more time then, please inform mentors and supervisors.
* If you have two previously assigned issues on Android-Kotlin pending then please first complete them.
> Supervisor @rahulsain

### PR DESCRIPTION
**Pull request by mrkc2303**
<hr>

## Description 📜
 
Fixes #5860 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Android Dev (Kotlin)

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by mrkc2303** [comment]
@rahulsain Kindly Review!
Thanks in Advance

**Comment by mrkc2303** [comment]
Hey @rahulsain 
Kindly review this PR.
Thanks

File: Android_Development_with_Kotlin/09. Dynamic User Interface/09.29 SearchView.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+### Step 2
+
+Add the following code to res/layout/activity_main.xml.
+
+```
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+   xmlns:tools="http://schemas.android.com/tools"
+   android:layout_width="match_parent"![143998097-71fb0b27-d488-4cf4-8d83-fbdb7acfd27a](https://user-images.githubusercontent.com/67221487/143998442-ec75d59a-2605-477b-8856-4ccf9409af11.png)
```
Comment (rahulsain) [inline Android_Development_with_Kotlin/09. Dynamic User Interface/09.29 SearchView.md:22]: you are displaying image in the codeview and only text are showing , please fix this

**Comment by mrkc2303** [comment]
@rahulsain Issue Solved, that was added mistakenly. 
Kindly check now

**Pull request merged**